### DISCUSSION TITLE
Running quads selectively

### DISCUSSION THREAD

**Discussion by Abhinandan-Krishnan** [USC-ACTLab/crazyswarm#776] [General]
Hello,

I wanted to run quads selectively. For example, if I  have 4 quads enabled on the test bed, let's say `cf1,cf2,cf3 and cf4` I want to take off just cf1 and cf2. How can I do it? 
Can I somehow use `groupMask` here?

Thank you!

**Comment by whoenig** [comment]
Yes, `groups` are the correct way, see https://github.com/USC-ACTLab/crazyswarm/blob/master/ros_ws/src/crazyswarm/scripts/pycrazyswarm/crazyflie.py#L121-L143 for some documentation on it.

**Reply by Abhinandan-Krishnan** [reply]
Thank you for the reply, so should I create `crazyflie` object, set the mask and then if I call takeoff on allcfs with a group mask, only the crazyflies with that group mask execute that take off?
